packages/sdk-react/src/components/payables/PayablesTable/PayablesTable.tsx

```diff
@@ -271,7 +277,24 @@ const PayablesTableBase = ({
         }),
         display: 'flex',
         width: 160,
-        renderCell: (params) => <PayableStatusChip status={params.value} />,
+        renderCell: (params) => {
+          const payable = params.row;
+          const isOverdue = isInvoiceOverdue(payable);
+
+          return (
+            <Box display="flex" alignItems="center" gap={1}>
+              <PayableStatusChip status={params.value} />
+              {isOverdue && (
+                <StyledChip
+                  className="Monite-PayableStatusChip"
```

Comment:
This code makes styling of that particular chip impossible: there is no way to apply a CSS style to overdue chip but not affect the status chip.
I see two possible solutions:
1. Expensive. We make a custom component like PayableStatusChip with all corresponding .stories.tsx files.
2. Cheap, but ugly: we add addition class Monite-PayableStatusChip-Overdue to allow theming this particular chip. This won't allow to change chip size, for example.

Reply:
If you choose the second approach, add a TODO comment explaining the necessary change, please.

Reply:
The scope of this PR was to add the ability to have a label, we can agree to put a ToDo mark as you suggested